package.json

```diff
@@ -36,7 +37,6 @@
     "devDependencies": {
         "@auto-it/conventional-commits": "^10.33.0",
         "@graphql-codegen/testing": "^1.17.7",
-        "@types/faker": "^5.5.9",
```

Comment:
It was probably left in there because it was forgotten to erase it, but now the type definitions have been imported into the repository and are no longer needed.
https://github.com/ardeois/graphql-codegen-typescript-mock-data/pull/93/commits/5a8cd1b094ffdd73277dc98a83cfdf50b52621d5